internal/testing/runchecks.sh

```diff
@@ -81,53 +90,65 @@ else
   exit $result
 fi
 
-# Ensure .go files are formatted with "gofmt -s".
+
+echo
+echo "Ensuring .go files are formatted with gofmt -s..."
 DIFF=$(gofmt -s -d `find . -name '*.go' -type f`)
 if [ -n "$DIFF" ]; then
-  echo "Please run gofmt -s and commit the result"
+  echo "FAIL: please run gofmt -s and commit the result"
   echo "$DIFF";
   exit 1;
 fi;
 
-# Ensure that the code has no extra dependencies (including transitive
-# dependencies) that we're not already aware of by comparing with
-# ./internal/testing/alldeps
-#
-# Whenever project dependencies change, rerun ./internal/testing/listdeps.sh
-#
-# Run this only on the latest version of Go, since module behavior may differ
-# across versions.
+
+echo
+echo "Ensuring that there are no dependencies not listed in ./internal/testing/alldeps..."
 if [[ $(go version) == *1\.12* ]]; then
   ./internal/testing/listdeps.sh | diff ./internal/testing/alldeps - || {
-    echo "FAIL: dependencies changed; compare listdeps.sh output with alldeps" && result=1
+    echo "FAIL: dependencies changed; run: internal/testing/listdeps.sh > internal/testing/alldeps" && result=1
+    # Module behavior may differ across versions.
+    echo "using go version 1.12."
   }
 fi
 
-# Ensure that any new packages have the corresponding entries in Hugo.
+
+echo
+echo "Ensuring that any new packages have the corresponding entries in Hugo..."
 missing_packages="$(internal/website/listnewpkgs.sh)"
 if ! [[ -z "$missing_packages" ]]; then
   echo "FAIL: missing package meta tags for:" 1>&2
   echo "$missing_packages" 1>&2
   result=1
 fi
 
-# Ensure that all examples used in Hugo match what's in source.
+
+echo
+echo "Ensuring that all examples used in Hugo match what's in source..."
 internal/website/gatherexamples/run.sh | diff internal/website/data/examples.json - > /dev/null || {
   echo "FAIL: examples changed; run: internal/website/gatherexamples/run.sh > internal/website/data/examples.json"
+  result=1
```

Comment:
Pretty sure this was a bug before, missing `result=1` meant that this wouldn't actual fail.